File: python/4/34.py
from functions4 import *

for sentence in morphological():
    for gram in ngram(sentence, 3):
        first, second, third = gram
        if second["surface"] == "の" and first["pos"] == third["pos"] == "名詞":
            print(first["surface"] + second["surface"] + third["surface"])

File: ruby/4/34.rb
require './util'

morphologies.each do |sentence|
  sentence.ngram(3).each do |(first, second, third)|
    if second[:surface] == "の" && first[:pos] == "名詞" && third[:pos] == "名詞"
      puts [first, second, third].map(&->word { word[:surface] }).join
    end
  end
end
